frontend/src/metabase/core/components/ColorPill/ColorPill.styled.tsx

```diff
@@ -1,9 +1,12 @@
 import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+
 import { color } from "metabase/lib/colors";
 
 export interface ColorPillRootProps {
   isAuto: boolean;
   isSelected: boolean;
+  pillSize: string;
```

Comment:
nit: you could use `PillSize` as the type which will provide autocompletion in this file.